.github/workflows/proto-upgrade.yml

```diff
@@ -0,0 +1,52 @@
+on:
+  workflow_dispatch:
```

Comment:
if this works we could use the `release` trigger to have this automatically run after Flipt release is created

Reply:
I like that idea. Does is work cross-repo? I remember having issues with other cross repo-actions.

In the end, we created a github app and used an action to use the app credentials to create an access token which we then used to trigger an action on another repo. Was a rube goldberg machine, but it worked.

Reply:
yeah, you cant use the provided `secrets.GITHUB_TOKEN` when doing triggers on release from other repos unfortunately, or even your same repo I think. something about preventing infinite loops. but we can use a PAT as an org secret. I already have one that can create releases

Reply:
also i think its worth keeping the `workflow_dispatch` trigger around too like you have it, so we can manually release if necessary